index.rst

```diff
@@ -187,15 +187,17 @@ Now that we've defined the skymap (formally the ``deepCoadd_skyMap`` data produc
 
   coaddDriver.py DATA --rerun example2 \
     --selectId visit=903334..903338:2 --selectId visit=903342..903346:2 \
-    --id tract=0 patch=1,1 filter=HSC-R --cores=4
+    --id tract=0 patch=1,1 filter=HSC-R --cores=4 \
+    --config assembleCoadd.doApplyUberCal=False makeCoaddTempExp.doApplyUberCal=False
 
 .. prompt:: bash
 
   coaddDriver.py DATA --rerun example2 \
     --selectId visit=903986..903990:2 --selectId visit=904010^904014 \
-    --id tract=0 patch=1,1 filter=HSC-I --cores=4
+    --id tract=0 patch=1,1 filter=HSC-I --cores=4 \
+    --config assembleCoadd.doApplyUberCal=False makeCoaddTempExp.doApplyUberCal=False
 
```

Comment:
Just FYI, it's only in `makeCoaddTempExp` that this config has any effect, but both it and `assembleCoaddTask` inherit from `coaddBaseTask` config, which is where this config is defined, so they both get a `doApplyUberCal` in their configs.  To avoid confusion, I do think it's good practice to set them both to the same value, so this is good :)